pkg/clients/acm/certificate.go

```diff
@@ -74,6 +74,21 @@ func GenerateCreateCertificateInput(name string, p *v1alpha1.CertificateParamete
 
 // GenerateCertificateStatus is used to produce CertificateExternalStatus from acm.certificateStatus
 func GenerateCertificateStatus(certificate acm.CertificateDetail) v1alpha1.CertificateExternalStatus {
+
```

Comment:
There is a test for this function: https://github.com/crossplane/provider-aws/blob/6dce815528e3df171052a7638935662ac6082ae9/pkg/clients/acm/certificate_test.go#L139-L174

Could you please add coverage for this branch? Thanks!

Reply:
@chlunde  thanks for review - i will check it